stable/slackin/values.yaml

```diff
@@ -0,0 +1,55 @@
+image:
+  name: chk1/slackin
+  digest: sha256:476ae1beb981449c1b61794bb9ec52beeff1c312ff54fbcd3010946b991975f8
```

Comment:
The `chk1/slackin` image does not tag releases.
If this is a problem, then I can create and maintain a slackin image myself.